### DISCUSSION TITLE
Cannot verify student status to sign up for copilot

### DISCUSSION THREAD

**Discussion by alanclayrichard** [community/community#49932] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I am trying to make my account a student account by adding my school email. Once the email is added and I try to sign up for student benefits, I select my school email, input my school name and then click continue but the page hangs after the continue button is clicked. When I refresh the page it starts over and I run into the same issue again. Any help would be appreciated!

**Comment by maahaMaya** [comment]
Most probably due more traffic on student account GitHub is not validate the account.
May you try next month.
